mne/viz/ica.py

```diff
@@ -206,11 +211,14 @@ def _plot_ica_sources_evoked(evoked, exclude, title, show):
     times = evoked.times * 1e3
 
     # plot unclassified sources
-    plt.plot(times, evoked.data.T, 'k')
+    if picks is None:
+        picks = np.arange(evoked.data.shape[0])
+    plt.plot(times, evoked.data[picks].T, 'k')
     for ii in exclude:
-        # use indexing to expose event related sources
-        plt.plot(times, evoked.data[ii].T, color='r', label='ICA %02d' % ii)
-
+        if ii in picks:
+            # use indexing to expose event related sources
+            plt.plot(times, evoked.data[ii].T, color='r',
+                     label='ICA %03d' % (ii + 1))
```

Comment:
BTW in your next PR can you also change these to `ax.plot` and `ax.set_xlabel` calls? I think that's the preferred way to do it instead of using the global-level `plt.plot`